apps/native/app/ios/Podfile

```diff
@@ -74,8 +74,23 @@ target 'IslandApp' do
     )
     __apply_Xcode_12_5_M1_post_install_workaround(installer)
     installer.pods_project.targets.each do |target|
+      if target.name == 'Flipper'
```

Comment:
note: react-native is ditching flipper, this is a workaround until flippers adds this include